### PR TITLE
External cvoc

### PR DESCRIPTION
**Pull request by ekoi**
**What this PR does / why we need it**:
 - Add Vocabulary URL field
 - Add parent term url
 - Min search letter 1
 - Add scrollbar for long list

**Which issue(s) this PR closes**:

Closes #

**Special notes for your reviewer**:

**Suggestions on how to test this**:

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**:

**Is there a release notes update needed for this change?**:

**Additional documentation**:

Screenshot 2021-02-17 at 13 31 34

### CODE REVIEW

File: src/main/java/edu/harvard/iq/dataverse/settings/SettingsServiceBean.java [hunk 1]
```diff
@@ -192,14 +190,19 @@
...
          * A link to an installation of https://github.com/IQSS/miniverse or
          * some other metrics app.
          */
         MetricsUrl,
-        
+
+        /**
+         * Key for Setting of Signposting
+         */
+        SignpostingConf,
```
Comment (PaulBoon) [inline src/main/java/edu/harvard/iq/dataverse/settings/SettingsServiceBean.java]: Strange, I expected this in the 'signposting' feature branch, somehow it now is also here?

Comment (PaulBoon) [reply src/main/java/edu/harvard/iq/dataverse/settings/SettingsServiceBean.java]: According to the annotations you added those lines today, and I don't see it being used.

**Review by PaulBoon — CHANGES_REQUESTED** [review]
Could you remove the SignpostingConf, because it should be in that other feature branch and it might result in a merge conflict

File: src/main/java/edu/harvard/iq/dataverse/settings/SettingsServiceBean.java (hunk 1 above)
Comment (ekoi) [reply src/main/java/edu/harvard/iq/dataverse/settings/SettingsServiceBean.java]: done.

**Pull request merged**